core/domain/collection_services_test.py

```diff
@@ -713,7 +755,22 @@ def test_get_collection_summaries_subscribed_to(self):
         self.assertEqual(summaries[4].title, 'Learning basic verbs in Spanish')
         self.assertEqual(summaries[5].title, 'Private collection in Spanish')
 
-    def test_get_collection_summaries_with_no_query(self):
+    def test_get_collection_summaries_with_no_query_dash_dash(self) -> None:
```

Comment:
I think we need to change the test name `test_get_collection_summaries_with_no_query_dash_dash` to a more understandable one.

Reply:
Changed!